### PR TITLE
Add test-related Email format

### PR DESCRIPTION
**Pull request by loay**

### CODE REVIEW

**Comment by davidcheung** [comment]
can u add a little explanation why `updated@bar.com` is bad?

**Comment by jannyHou** [comment]
tbh I am ok with any standard, as long as we **have** a standard lol

File: style-guide.md [hunk 1]
```diff
@@ -230,6 +230,21 @@ done();
 - Do not use directories like `/tmp/sandbox` as you will run into
   permission issues on CI for directories that are not in the project
 
+## Email examples
+
+- All test-related email examples should be of the format `email@example.com`.
+- The `example.com` domain is created to be used for examples in documents, and could be used without prior coordination or asking for permission.
```
Comment (bajtos) [inline style-guide.md]: The `example.come` domain *was* created?

**Comment by bajtos** [comment]
One nit-pick, the rest LGTM 👍

**Pull request merged**